bugbug/repository.py

```diff
@@ -709,14 +719,20 @@ def download_commits(repo_dir, rev_start=0, ret=False, save=True):
         commits = executor.map(_transform, commits, chunksize=64)
         commits = tqdm(commits, total=commits_num)
 
-        if ret:
-            commits = list(commits)
+    commits = list(commits)
 
-        if save:
-            db.append(COMMITS_DB, commits)
+    calculate_experiences(commits, first_pushdate, save)
+
+    if ret:
```

Comment:
Since you're collecting commits to `list` anyway, I'd remove `ret` completely and consider it as always True